src/components/interfaces/MOBILE_API.xml

```diff
@@ -5003,6 +5032,36 @@
     </param> 
   </function>
 
+  <function name="SendHapticData" functionID="SendHapticDataID" messagetype="request" >

```

Comment:
Without SendHapticData being added to the `sdl_preloaded_pt.json`, it needs to be added manually by anyone wanting to use this feature. It should at least exist in one of the functional groups, though I'm not certain if it would fit in any existing one. It may need its own.

Reply:
As soon as this is fixed, I should be able to approve these changes

Reply:
I will add the group to functional_groupings.

```
"HapticGroup": {
  "rpcs": {
    "OnTouchEvent": {
      "hmi_levels": ["FULL"]
    }
    "SendHapticData": {
      "hmi_levels": ["FULL"]
    }
  }
}
```

Does this also require discussion with SDL proposal?

Reply:
I believe that we should include this information in any proposal introducing a new RPC in the future, but I don't see it being worth the hassle to add another proposal amendment for this change now. I believe that we can improvise in this case.

I intend to improve the guidelines in the SDL evolution repository in the near future for any proposals involving RPC changes/additions, which will hopefully fix this issue in the future.